### PR TITLE
en,zh: release notes for 1.5.0-beta.1

### PR DESCRIPTION
**Pull request by KanShiori**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.5.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- Allow users to define strategy to restart failed backup jobs, increasing backup stability ([#4883](https://github.com/pingcap/tidb-operator/pull/4883), [@WizardXiao](https://github.com/WizardXiao)) ([#4925](https://github.com/pingcap/tidb-operator/pull/4925), [@WizardXiao](https://github.com/WizardXiao))
+
+## Improvements
+
+- Upgraded Kubernetes dependencies to version v1.20 ([#4954](https://github.com/pingcap/tidb-operator/pull/4954), [@KanShiori](https://github.com/KanShiori))
+
+- Add the metrics for the reconciler and worker queue to improve observability ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+
+- When rolling upgrade TiKV Pods, TiDB Operator will wait for the leader to transfer back to the upgraded Pod before upgrading the next TiKV node. This helps to reduce performance degradation during rolling upgrade ([#4863](https://github.com/pingcap/tidb-operator/pull/4863), [@KanShiori](https://github.com/KanShiori))
```
Comment (csuzhangxc) [inline en/releases/release-1.5.0-beta.1.md:29]:
```suggestion
- When rolling upgrade TiKV Pods, TiDB Operator will wait for the leader to transfer back to the upgraded Pod before upgrading the next TiKV pod. This helps to reduce performance degradation during rolling upgrade ([#4863](https://github.com/pingcap/tidb-operator/pull/4863), [@KanShiori](https://github.com/KanShiori))
```

as we use `TiKV Pods` at the beginning.

File: zh/releases/release-1.5.0-beta.1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- 允许用户自定义 Prometheus Scraping 相关配置 ([#4846](https://github.com/pingcap/tidb-operator/pull/4846), [@coderplay](https://github.com/coderplay))
+
+- TiProxy 支持共享部分 TiDB 的证书 ([#4880](https://github.com/pingcap/tidb-operator/pull/4880), [@xhebox](https://github.com/xhebox))
+
+- 当配置 `spec.preferIPv6` 为 `true` 时，为所有组件的 Service 的 `ipFamilyPolicy` 配置为 `PreferDualStack` ([#4959](https://github.com/pingcap/tidb-operator/pull/4959), [@KanShiori](https://github.com/KanShiori))
+
+## Bug 修复
+
+- 修复由于与 metric 接口冲突导致 pprof 接口不可访问的问题 ([#4874](https://github.com/pingcap/tidb-operator/pull/4874), [@hanlins](https://github.com/hanlins))
```
Comment (ran-huang) [inline zh/releases/release-1.5.0-beta.1.md:39]:
```suggestion
- 修复了因为 metric 接口冲突而导致 pprof 接口无法访问的问题 ([#4874](https://github.com/pingcap/tidb-operator/pull/4874), [@hanlins](https://github.com/hanlins))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## New Feature
+
+- Support using the `tidb.pingcap.com/pd-transfer-leader` annotation to restart PD Pod gracefully ([#4896](https://github.com/pingcap/tidb-operator/pull/4896), [@luohao](https://github.com/luohao))
+
+- Support managing TiCDC with Advanced StatefulSet ([#4881](https://github.com/pingcap/tidb-operator/pull/4881), [@charleszheng44](https://github.com/charleszheng44))
+
+- Support managing TiProxy with Advanced StatefulSet ([#4917](https://github.com/pingcap/tidb-operator/pull/4917), [@xhebox](https://github.com/xhebox))
+
+- Add a new field `bootstrapSQLConfigMapName` in the TiDB spec to specify an initialize sql file on TiDB's first bootstrap ([#4862](https://github.com/pingcap/tidb-operator/pull/4862), [@fgksgf](https://github.com/fgksgf))
```
Comment (ran-huang) [inline en/releases/release-1.5.0-beta.1.md:19]:
```suggestion
- Add a new field `bootstrapSQLConfigMapName` in the TiDB spec to specify an initialization SQL file on TiDB's first bootstrap ([#4862](https://github.com/pingcap/tidb-operator/pull/4862), [@fgksgf](https://github.com/fgksgf))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+- Allow users to define strategy to restart failed backup jobs, increasing backup stability ([#4883](https://github.com/pingcap/tidb-operator/pull/4883), [@WizardXiao](https://github.com/WizardXiao)) ([#4925](https://github.com/pingcap/tidb-operator/pull/4925), [@WizardXiao](https://github.com/WizardXiao))
```
Comment (ran-huang) [inline en/releases/release-1.5.0-beta.1.md:21]:
```suggestion
- Allow users to define a strategy to restart failed backup jobs, enhancing backup stability ([#4883](https://github.com/pingcap/tidb-operator/pull/4883), [@WizardXiao](https://github.com/WizardXiao)) ([#4925](https://github.com/pingcap/tidb-operator/pull/4925), [@WizardXiao](https://github.com/WizardXiao))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Improvements
+
+- Upgraded Kubernetes dependencies to version v1.20 ([#4954](https://github.com/pingcap/tidb-operator/pull/4954), [@KanShiori](https://github.com/KanShiori))
```
Comment (ran-huang) [inline en/releases/release-1.5.0-beta.1.md:25]:
```suggestion
- Upgrade Kubernetes dependencies to v1.20 ([#4954](https://github.com/pingcap/tidb-operator/pull/4954), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+- Add the metrics for the reconciler and worker queue to improve observability ([#4882](https://github.com/pingcap/tidb-operator/pull/4882), [@hanlins](https://github.com/hanlins))
+
+- When rolling upgrade TiKV Pods, TiDB Operator will wait for the leader to transfer back to the upgraded Pod before upgrading the next TiKV node. This helps to reduce performance degradation during rolling upgrade ([#4863](https://github.com/pingcap/tidb-operator/pull/4863), [@KanShiori](https://github.com/KanShiori))
+
+- Allow users to customize Prometheus scraping settings ([#4846](https://github.com/pingcap/tidb-operator/pull/4846), [@coderplay](https://github.com/coderplay))
+
+- Support for sharing some TiDB's certificates with TiProxy ([#4880](https://github.com/pingcap/tidb-operator/pull/4880), [@xhebox](https://github.com/xhebox))
```
Comment (ran-huang) [inline en/releases/release-1.5.0-beta.1.md:33]:
```suggestion
- Support sharing some of TiDB's certificates with TiProxy ([#4880](https://github.com/pingcap/tidb-operator/pull/4880), [@xhebox](https://github.com/xhebox))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+- Configure the field `ipFamilyPolicy` as `PreferDualStack` for all components' Services when `spec.preferIPv6` is `true` ([#4959](https://github.com/pingcap/tidb-operator/pull/4959), [@KanShiori](https://github.com/KanShiori))
```
Comment (ran-huang) [inline en/releases/release-1.5.0-beta.1.md:35]:
```suggestion
- Configure the field `ipFamilyPolicy` as `PreferDualStack` for all components' Services when `spec.preferIPv6` is set to `true` ([#4959](https://github.com/pingcap/tidb-operator/pull/4959), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.5.0-beta.1.md [hunk 8]
```diff
@@ -20,6 +20,8 @@ TiDB Operator version: 1.5.0-beta.1
 
 - Allow users to define a strategy to restart failed backup jobs, enhancing backup stability ([#4883](https://github.com/pingcap/tidb-operator/pull/4883), [@WizardXiao](https://github.com/WizardXiao)) ([#4925](https://github.com/pingcap/tidb-operator/pull/4925), [@WizardXiao](https://github.com/WizardXiao))
 
+- Support using the `tidb.pingcap.com/tidb-graceful-shutdown` annotation to restart TiDB Pod gracefully ([#4948](https://github.com/pingcap/tidb-operator/pull/4948), [@wxiaomou](https://github.com/wxiaomou))
```
[the remark this answers is not in the document]
Comment (KanShiori) [reply en/releases/release-1.5.0-beta.1.md]: done

**Review by ran-huang — APPROVED** [review]

**Pull request merged**